packages/twenty-server/src/engine/core-modules/exception-handler/drivers/sentry.driver.ts

```diff
@@ -27,6 +27,14 @@ export class ExceptionHandlerSentryDriver
         scope.setExtra('workspace', options.workspace);
       }
 
+      if (options?.messageChannel) {
+        scope.setExtra('messageChannel', options.messageChannel);
```

Comment:
messageChannel and calendarChannel are business features and should not be part of an engine module.

It's interesting to pass additional information but could we pass something more generic?
scope.setExtra('additionalInfo');